pages/_app.tsx

```diff
@@ -66,6 +66,13 @@ export default function App({ Component, pageProps }: AppProps) {
           <a style={{ marginRight: 15 }}>Example 5</a>
         </LinkIfNotCurrent>
 
+        <LinkIfNotCurrent
+          href="/example-6-xss/vulnerable"
+          baseHref="/example-6-xss/"
+        >
+          <a style={{ marginRight: 15 }}>Example 6</a>
+        </LinkIfNotCurrent>
```

Comment:
we also need a link on the index page

I'll take this over

Reply:
Oh, this was already done in 8609ea82d219292ea4a5e01ed5352537a93f125b, resolving this